src/Symfony/Component/Security/Core/Resources/translations/security.id.xlf

```diff
@@ -4,7 +4,7 @@
         <body>
             <trans-unit id="1">
                 <source>An authentication exception occurred.</source>
-                <target>Terjadi sebuah pengecualian otentikasi.</target>
+                <target>Terjadi kesalahan otentikasi.</target>
```

Comment:
I propose this alternative. It literally means "An authentication error occurred.". Since "pengecualian" (exception) here is too generic and ambiguous, it's rarely used in apps.